.circleci/config.yml

```diff
@@ -14,15 +14,15 @@ jobs:
     steps:
       - checkout
       - persist_to_workspace:
-          root: /go
+          root: ~/
           paths:
-            - src/github.com/sylabs/singularity-cri
+            - singularity-cri
 
   gometalinter_check:
     <<: *defaults
     steps:
       - attach_workspace:
-          at: /go
+          at: ~/
       - run:
           name: Install golangci-lint v1.15
```

Comment:
Still needs a config file for golanglint-ci that has the following content:

```
[vagrant@localhost singularity-cri]$ cat .golangci.yml
run:
  modules-download-mode: vendor
```

This also requires a newer golangci-lint. I tested with v1.17.1, the latest release.

Reply:
I don't understand how this will fix the error. The `go.mod` fix should fix linter as well...

Reply:
But updating linter is in general nice idea!